HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -204,11 +204,22 @@ def customiseFor2018Input(process):
 
     return process
 
+def customiseFor33104(process):
+    """Customise the HLT menu to remove the unused use_vdt parameters"""
+    for producer in producers_by_type(process, "PrimaryVertexProducer"):
+        if hasattr(producer, "TkClusParameters"):
+            pset = getattr(producer, "TkClusParameters")
+            if hasattr(pset, "TkDAClusParameters"):
+                del producer.TkClusParameters.TkDAClusParameters.use_vdt
```

Comment:
Could you please delete only after checking that ```use_vdt``` exists? That would allow to decouple removing the parameter from the configs in ConfDB, and removing this customisation.

Reply:
@Martin-Grunewald, I could but that would be redundant. 
As far as I can tell the parameter `use_vdt` always exists provided the mother `PSet` `TkDAClusParameters` exists.
If that would have not been the case, this branch would have failed the integration tests, as deleting a non-existent parameter results in a python compilation time error (cf. [here](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20c809/13372/runTheMatrixINPUT-results/158.1_QCD_Pt_80_120_13_HI+QCD_Pt_80_120_13_HIINPUT+DIGIHI2018PPRECO+RECOHI2018PPRECO+HARVESTHI2018PPRECO/step2_QCD_Pt_80_120_13_HI+QCD_Pt_80_120_13_HIINPUT+DIGIHI2018PPRECO+RECOHI2018PPRECO+HARVESTHI2018PPRECO.log) )

Reply:
Yes. But now the parameter needs to be removed. This is a multi-step procedure for me as I need to remove it in the menus in ConfDB, and then make a PR with the new menus and the above customisation removed. In the mean time YSG developers may take the modified menu from ConfDB and a release with this PR, but before I make the PR with updated dumps and this customisation removed. In that case, with the parameter removed in the menu itself and then still to be deleted with the above customisation, there will be an error.

Reply:
@Martin-Grunewald thanks for the explanation.
I have updated the branch with:
```diff
+def customiseFor33104(process):
+    """Customise the HLT menu to remove the unused use_vdt parameters"""
+    for producer in producers_by_type(process, "PrimaryVertexProducer"):
+        if hasattr(producer, "TkClusParameters"):
+            pset = getattr(producer, "TkClusParameters")
+            if hasattr(pset, "TkDAClusParameters"):
+                if hasattr(getattr(pset, "TkDAClusParameters"),"use_vdt"):
+                    del producer.TkClusParameters.TkDAClusParameters.use_vdt
+
+    return process
```
let me know if there is a more elegant way.

Reply:
Thanks a lot!